src/store/slices/account-slice.ts

```diff
@@ -77,8 +83,15 @@ export const loadAccountDetails = createAsyncThunk("account/loadAccountDetails",
         unstakeAllowance = await ssbContract.allowance(address, addresses.STAKING_ADDRESS);
     }
 
+    if (addresses.WSSB_ADDRESS) {
+        const wssbContract = new ethers.Contract(addresses.WSSB_ADDRESS, MemoTokenContract, provider);
+        wssbBalance = await wssbContract.balanceOf(address);
+        // unstakeAllowance = await wssbContract.allowance(address, addresses.STAKING_ADDRESS);
```

Comment:
Shouldn't we remove this commented assignation?

Reply:
if you want to keep it for later usage, in that case could you please add a comment below that commented line to explain why it's there

Reply:
oh, i actually didn't know what that did.  I had copied that section from the previous ones and modified, accordingly.  I'll just remove it